<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugSelectCirc</class>
 <widget class="QWidget" name="DebugSelectCirc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Load">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>加载图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Pre">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>预览模式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Edit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>编辑模式</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="DebugSelectView" name="graphicsView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DebugSelectView</class>
   <extends>QGraphicsView</extends>
   <header>DebugSelectView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
